test/unit/player.test.js

```diff
@@ -2209,6 +2209,29 @@ QUnit.test('should not allow to register custom player when any player has been
   videojs.registerComponent('Player', Player);
 });
 
+QUnit.test('setters getters passed to tech', function(assert) {
```

Comment:
This test is unrelated to the changes but makes Codecov happy as it increases coverage...